changelog/392.doc.rst

```diff
@@ -0,0 +1 @@
+Split up documentation into multiple files.
```

Comment:
This needs to be longer, explain what the changes are more in depth. Making comment now so its not forgotten.

Reply:
Done in [4cc42e0f32291197952ae33f3572ea2a19196a58](https://github.com/DisnakeDev/disnake/pull/722/commits/4cc42e0f32291197952ae33f3572ea2a19196a58)